ops/ansible/playbooks-healthapt/backend.yml

```diff
@@ -99,8 +97,7 @@
         data_server_appserver_management_username: "{{ vault_data_server_appserver_management_username }}"
         data_server_appserver_management_password: "{{ vault_data_server_appserver_management_password }}"
         data_server_appserver_https_port: "{{ vault_data_server_appserver_https_port }}"
-        data_server_war_name: "bfd-server-war-{{ data_server_version }}.war"
-        data_server_war_local_dir: "~/.m2/repository/gov/cms/bfd/bfd-server-war/{{ data_server_version }}"
+        # Note: the `data_server_war` variable should have been provided via `--extra-vars` on the command line.
```

Comment:
noted, will need update packer ansible provisioner for pipeline and server to include these.

Reply:
Please note that these paths will be passed in via the `AppBuildResults` instance that gets passed to the Groovy method.